src/common.c

```diff
@@ -653,16 +653,16 @@ static void fft_g1_fast(
  * @param[in]   s   The trusted setup
  *
  * @remark The array lengths must be a power of two.
- * @remark Use ifft_g1() for inverse transformation.
+ * @remark Use g1_ifft() for inverse transformation.
  */
-C_KZG_RET fft_g1(g1_t *out, const g1_t *in, size_t n, const KZGSettings *s) {
+C_KZG_RET g1_fft(g1_t *out, const g1_t *in, size_t n, const KZGSettings *s) {
```

Comment:
Why are we renaming these functions? The spec also names them `fft_field` and `coset_fft_field` (the type is in the suffix).

Reply:
In C, the type is the prefix. For example:

![telegram-cloud-photo-size-1-4994634266591735274-y](https://github.com/user-attachments/assets/f2f5d533-b277-47ca-a928-1e8b9dcd81bf)